instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/patches/active_job_callbacks.rb

```diff
@@ -72,6 +73,10 @@ def job_attributes(job)
               'messaging.destination' => job.queue_name,
               'messaging.message_id' => job.job_id,
               'messaging.active_job.provider_job_id' => job.provider_job_id,
+              'messaging.active_job.enqueued_at' => job.enqueued_at,
+              'messaging.active_job.enqueued_duration_ms' => milliseconds_since(job.enqueued_at),
+              'messaging.active_job.first_enqueued_at' => job.first_enqueued_at,
+              'messaging.active_job.enqueued_and_retry_duration_ms' => milliseconds_since(job.first_enqueued_at),
```

Comment:
I have **no** strong opinion on the naming of `enqueued_duration_ms` or `enqueued_and_retry_duration_ms`.  My other thought was `enqueued_wait_ms` and `enqueued_and_retry_wait_ms`.  I chose `duration` because OTel already captures a `duration_ms` metric for spans, so I opted for consistency with the naming here.